src/simtools/simulator.py

```diff
@@ -114,16 +114,27 @@ def _initialize_array_model(self, mongo_db_config):
 
         Returns
         -------
-        ArrayModel
-            ArrayModel object.
-        """
-        return ArrayModel(
-            label=self.label,
-            site=self.args_dict.get("site"),
-            layout_name=self.args_dict.get("array_layout_name"),
-            mongo_db_config=mongo_db_config,
-            model_version=self.args_dict.get("model_version", None),
-        )
+        list
```

Comment:
To be consistent in naming, change this to `_initialize_array_models`

Reply:
Done.